src/server/graphql/models/Team/createFirstTeam/addSeedProjects.js

```diff
@@ -15,15 +15,15 @@ const SEED_PROJECTS = [
     status: ACTIVE,
     sortOrder: 1,
     content: `
-      Try a test run of an Action meeting. Tap on ‘Meeting Lobby’ in
```

Comment:
OOoo, this is a tricky one. This isn't referring to Action, but an _Action Meeting_-- a special kind of meeting.  I kinda think we should keep this for the holacracy crowd AHA moment.